dotCMS/src/main/java/com/dotcms/services/VanityUrlServices.java

```diff
@@ -216,11 +214,13 @@ public void onCommitListener(final CommitListenerEvent commitListenerEvent) {
                                 "Unable to invalidate VanityURL in cache:" +
                                         contentlet));
             }
+        } catch (NotFoundInDbException e) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Either log or rethrow this exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1166)